community/Brian/index.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
```

Comment:
Again, your indentation across the html and css files here are almost double. One indent should equal roughly 4 spaces (2 spaces if you are that way inclined) but no more than 4. At the moment, this might appear to be your build in Prettier/Formatter/Linter causing the aggressive indentation? Can you take a look at this too? 👍